doc/environment.yml

```diff
@@ -16,3 +16,4 @@ dependencies:
 - sphinx
 - sphinx_rtd_theme
 - ipython=4.0.1
+- cartopy
```

Comment:
sounds good - though FYI this one won't be updated as quickly as the `pip` version (e.g., conda-forge is out of date right now)